roboteam_ai/src/skills/Skill.h

```diff
@@ -6,15 +6,14 @@
 #include <roboteam_utils/Angle.h>
 #include "ros/ros.h"
 #include "../io/IOManager.h"
-#include "../control/positionControllers/PosVelAngle.h"
+#include "roboteam_ai/src/control/positionControllers/RobotCommand.h"
```

Comment:
Or, just include it in the relevant files? this smells like unnecessary compilation time since you're not changing the .cpp

Reply:
Agreed with rolf